.github/workflows/nix.yml

```diff
@@ -0,0 +1,76 @@
+name: "NixOS Builds"
+on:
+  pull_request:
+  push:
+jobs:
+  dev:
+    strategy:
+      matrix:
+        include:
+          - name: nixos-20.09
```

Comment:
Will this add 3 more builds to CI? We can't add builds for every possible distribution out there.